nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/command/PositionCommand.java

```diff
@@ -102,8 +102,9 @@ protected void toEnd(LogManager manager, String group, String name) {
 
         try (LogTailer<Record> tailer = manager.createTailer(group, name)) {
             tailer.toEnd();
+            tailer.commit();
         }
-        System.out.println(String.format("# Moved log %s, group: %s, from: %s to %s", name, group, lag.lag(), lag.upper()));
+        System.out.println(String.format("# Moved log %s, group: %s, from: %s to %s", name, group, lag.lower(), lag.upper()));
```

Comment:
Why are we using `System.out.println` ?

Reply:
This is a standalone command line tool so the output needs to go to the console.

Reply:
I see. 👍